src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/ApprenticeshipCommitmentsJobsHandler.cs

```diff
@@ -20,11 +23,31 @@ public async Task Handle(ApprenticeshipCreatedEvent message, IMessageHandlerCont
             }
             else
             {
-                await api.CreateApprentice(message.ToApprenticeshipCreated());
+                // create registration
+                var res = await api.CreateApprentice(message.ToApprenticeshipCreated());
+
+                var invite = new ls.SendInvitation()
+                {
+                    ClientId = Guid.Parse(res.ClientId),
+                    SourceId = res.SourceId.ToString(),                    
+                    GivenName = res.GivenName,
+                    FamilyName = res.FamilyName,
+                    Email = "mrchas@chastest.com",
+                    OrganisationName = message.LegalEntityName,
+                    ApprenticeshipName = res.ApprenticeshipName,
+                    Callback = res.CallbackUrl,
+                    UserRedirect = res.RedirectUrl,
+                };
```

Comment:
Not sure if the callback and redirectUrl values will be needed, but if they are it does make more sense for the config to be in the Jobs project

Reply:
Created https://skillsfundingagency.atlassian.net/browse/CS-777 to move the configs